lib/OpenQA/Schema.pm

```diff
@@ -136,6 +142,7 @@ sub create_system_user ($self) {
 
 # read application secret from database
 sub read_application_secrets ($self) {
+
```

Comment:
unrelated whitespace changes

Reply:
just wait. at least as far the `not-ready` flag in on) please